core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -106,12 +107,19 @@ public KafkaStore(SchemaRegistryConfig config,
     this.zkUtils = ZkUtils.apply(
         kafkaClusterZkUrl, zkSessionTimeoutMs, zkSessionTimeoutMs,
         JaasUtils.isZkSecurityEnabled());
-    this.kafkaTopicReader =
-        new KafkaStoreReaderThread<>(zkUtils, kafkaClusterZkUrl, topic, groupId,
-                                         Integer.MIN_VALUE, this.storeUpdateHandler,
-                                         serializer, this.localStore, this.noopKey);
     this.brokerSeq = zkUtils.getAllBrokersInCluster();
 
+    List<Broker> brokers = JavaConversions.seqAsJavaList(brokerSeq);
+    StringBuilder sb = new StringBuilder();
+    for (Broker broker : brokers) {
+      for(EndPoint ep : JavaConversions.asJavaCollection(broker.endPoints().values())) {
```

Comment:
Actually, I think a better approach would be to `log.warn` when a non-plaintext is found, and ignore it. And throw an exception if there are no plaintext endpoints.